acceptance/bundle/deploy/secret-scope/databricks.yml.tmpl

```diff
@@ -0,0 +1,13 @@
+bundle:
+  name: deploy-secret-scope-test-$UNIQUE_NAME
```

Comment:
question - if it's local-only, do you still need $UNIQUE_NAME suffix?

Reply:
this test run in the cloud as well